ddtrace/context.py

```diff
@@ -57,6 +57,8 @@ def __init__(
         self._meta = meta if meta is not None else {}  # type: _MetaDictType
         self._metrics = metrics if metrics is not None else {}  # type: _MetricDictType
 
+        self._meta["language"] = meta.language if (meta is not None and "language" in meta) else "python"
```

Comment:
```suggestion
        if "language" not in self._meta:
            self._meta["language"] = "python"
```

or maybe just having L57 as `self._meta = meta if meta is not None else {"language": "python"}` would also work?